snuba/migrations/snuba_migrations/transactions/0002_transactions_onpremise_fix_orderby_and_partitionby.py

```diff
@@ -64,6 +74,13 @@ def forwards() -> None:
             curr_partition_key, new_partition_key
         )
 
+    # Switch the primary key
+    if primary_key_needs_update:
+        assert new_create_table_statement.count(curr_partition_key) == 1
+        new_create_table_statement = new_create_table_statement.replace(
+            curr_primary_key, new_primary_key
+        )
+
```

Comment:
I would add the `ORDER BY` keyword to the clause you are trying to replace. So you do not replace by mistake the same list of columns potentially present somewhere else in the statement. Yes it is unlikely but does not hurt.

Reply:
There's an assertion that the list of columns only occurs once (it was wrong before, just fixed it). We do the same for replacing the partition key.